lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -292,11 +292,17 @@ def max_thread_limit
     end
     next if File::exists?(target)
 
+    # https://tickets.puppetlabs.com/browse/PUP-4884
+    working_dir = File.expand_path('spec/fixtures/work-dir')
+    working_dir = working_dir.gsub(/\\/, '/')
+    target_dir = File.expand_path('spec/fixtures/modules')
+    target_dir = target_dir.gsub(/\\/, '/')
+
     command = "puppet module install" + ref + flags + \
       " --ignore-dependencies" \
       " --force" \
-      " --module_working_dir spec/fixtures/module-working-dir" \
-      " --target-dir spec/fixtures/modules #{remote}"
+      " --module_working_dir #{working_dir}" \
+      " --target-dir #{target_dir} #{remote}"
```

Comment:
I should have mentioned it is not super easy to see what is going on, that is why this took longer to find. This change is going from specifying a relative path to specifying a full path. The problem with the relative path is that PUP-4884 doesn't expand the path properly and so it becomes something like `C:\somewhere/spec/fixtures/work-dir`, and then it fails.